### PR TITLE
Feat:  homepage added

### PR DESCRIPTION
**Pull request by aryan-13**
- Added navbar
- hero section
- offer section
- categories 
- footer

### CODE REVIEW

**Review by omgaikwad — COMMENTED** [review]
Code looks good, some files are in ".jfif" which aren't loading, I don't know what it represents, other than that everything seems fine.

File: css/layout/home-nav.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+.home-nav{
+    font-size:1.7rem;
+    width:100wh;
+    opacity: 0.9;
+   background-color: var(--color-primary);
+
+}
+.navigation-box{
+    padding: var(--size-4);
+    /* background-color: var(--color-primary); */
```
Comment (Hamdrive) [inline css/layout/home-nav.css]: remove comments

**Review by Hamdrive — COMMENTED** [review]
Looks good so far, suggestion to cleanup comments

File: css/layout/home-nav.css (hunk 1 above)
Comment (aryan-13) [reply css/layout/home-nav.css]: Done

**Pull request merged**